nixos/tests/all-tests.nix

```diff
@@ -1437,6 +1437,7 @@ in
   xterm = runTest ./xterm.nix;
   xxh = runTest ./xxh.nix;
   yabar = runTest ./yabar.nix;
+  yarr = handleTest ./yarr.nix { };
```

Comment:
It's better to use `runTest`, see https://github.com/NixOS/nixpkgs/issues/386873.